src/backend/api/meals.js

```diff
@@ -1,15 +1,130 @@
 const express = require("express");
+const app = require("../app");
+//const { max } = require("../database");
 const router = express.Router();
 const knex = require("../database");
 
+//Get meals that has a price smaller than maxPrice
 router.get("/", async (request, response) => {
+  console.log("akjdskank");
```

Comment:
It's great that you have console.logs to imply you tested your code but always remember to remove them as well.